presto-orc/src/main/java/com/facebook/presto/orc/ChunkedSliceOutput.java

```diff
@@ -365,6 +367,9 @@ public ChunkSupplier(int minChunkSize, int maxChunkSize)
         public void reset()
         {
             bufferPool.addAll(0, usedBuffers);
+            for (byte[] b : bufferPool) {
+                bufferPollSize += SizeOf.sizeOf(b);
```

Comment:
Can b.length be used here? b is not null.

Reply:
SizeOf.sizeOf(b)=(array == null)? 0: sizeOfByteArray(array.length)
sizeOfByteArray(array.length)=ARRAY_BYTE_BASE_OFFSET + (((long) ARRAY_BYTE_INDEX_SCALE) * length)

It seems that SizeOf.sizeOf(b) is more accurate than b.length.